.github/workflows/update-vcpkg-release.yml

```
@@ -0,0 +1,58 @@
+
+      - name: Update vcpkg builtin-baseline
+        run: |
+          ./external/vcpkg/bootstrap-vcpkg.sh
+          ./external/vcpkg/vcpkg x-update-baseline
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: chore/vcpkg-update-${{ steps.vcpkg_tag.outputs.tag }}
+          commit-message: chore(vcpkg): update to latest vcpkg release ${{ steps.vcpkg_tag.outputs.tag }}
```

actionlint (1.7.10)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)